documentation/book/ref-operators-cluster-operator-configuration.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+
+[id='ref-operators-cluster-operator-configuration-{context}']
+= Cluster Operator Configuration
+
+The Cluster Operator can be configured through environment variables.
```

Comment:
The Cluster Operator can be configured through environment variables.
The following environment variables are supported:

TO

The Cluster Operator can be configured through the following supported environment variables.